packages/oruga/src/components/steps/tests/steps.integration.test.ts

```diff
@@ -0,0 +1,59 @@
+import { afterEach, describe, expect, test } from "vitest";
+import { enableAutoUnmount, mount } from "@vue/test-utils";
+import { axe, toHaveNoViolations } from "jest-axe";
+
+import StepsExample from "./StepsExample.vue";
+
+expect.extend(toHaveNoViolations);
```

Comment:
Do we have to add this in every test file now? Or is it a one time think? If so, you can also add a `vitest.setup.ts` file in a `/src/__tests__/` folder and a reference to the setup file in `vite.config.ts` with `test:{ setupFiles: [require.resolve("./src/__tests__/vitest.setup.ts")],}`